mkdocs/themes/mkdocs/css/base.css

```
@@ -77,16 +88,17 @@ code.code-large, .rst-content tt.code-large {
 
 code {
     padding: 2px 5px;
-    background: #fff;
-    border: solid 1px #e1e4e5;
-    color: #333;
+    /* csslint ignore:start */
+    background-color: var(--bs-secondary-bg-subtle);
+    border: solid 1px var(--bs-secondary-border-subtle);
+    color: var(--bs-secondary-text-emphasis);
```

And let's match it for `code`.

**But** with subtler background because it's too noisy inside admonitions otherwise

```suggestion
    background-color: rgba(var(--bs-body-bg-rgb), 0.75);
    border: solid 1px var(--bs-border-color);
    color: var(--bs-body-color);
```

---

Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae